web/src/beta/features/SidePanel/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { CSSProperties, ReactNode } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+export type SidePanelContent = {
+  id: string;
+  title: ReactNode;
+  children: ReactNode;
+  maxHeight?: CSSProperties["maxHeight"];
+};
+type Props = {
+  location: "left" | "right";
+  contents: SidePanelContent[];
+};
+
+const SidePanel: React.FC<Props> = ({ location, contents }) => {
```

Comment:
Please keep your style the same. If you import FC directly, please always do that. If you use React.FC, please always do that

Reply:
Sorry...! I want to follow project styles (`React.FC<Props>`). Fixed in [here](https://github.com/reearth/reearth/pull/463/commits/8b7470a2039c593fcd01539bf2c9e34d6c59e633)